ddev/tests/conftest.py

```diff
@@ -71,7 +71,7 @@ def local_repo() -> Path:
 @pytest.fixture(scope='session')
 def valid_integrations(local_repo) -> list[str]:
     repo = Repository(local_repo.name, str(local_repo))
-    return [path.name for path in repo.integrations.iter_all()]
+    return [path.name for path in repo.integrations.iter_all(('all',))]
```

Comment:
Optional nit: you may avoid the awkward 1-element tuple syntax.

```suggestion
    return [path.name for path in repo.integrations.iter_all(['all'])]
```